NOnion/Utility/Base64Util.fs

```diff
@@ -21,3 +23,6 @@ module Base64Util =
 
     let FromString(input: string) =
         FixMissingPadding input |> System.Convert.FromBase64String
+
+    let EncodeNoPaddding(input: array<byte>) =
+        (Convert.ToBase64String input).TrimEnd([| '=' |])
```

Comment:
@aarani has an overload that accepts a single char, let's use it so that you don't need to wrap it in [| |] (and anyway everytime you want to wrap it like that we prefer to use singleton funcs)